### PR TITLE
[1.13] Gather metrics from Fluent Bit

### PR DESCRIPTION
**Pull request by philipnrmn**
## High-level description

This PR configures telegraf to gather metrics from Fluent Bit

## Corresponding DC/OS tickets (required)

  - [DCOS-54425](https://jira.mesosphere.com/browse/DCOS-54425) Configure Telegraf to gather metrics from Fluent Bit

### CODE REVIEW

**Comment by philipnrmn** [comment]
@branden and @gracedo , would you kindly take a look at this when you get a minute?

**Comment by philipnrmn** [comment]
We have smoke tests in place to make sure config changes didn't break things. To date, where we've changed the configuration but not the code, we haven't added integration tests, so I followed that pattern.

**Comment by gracedo** [comment]
I was thinking along the lines of adding a test to check if Fluent Bit metrics make it into the Prom metrics output - I think when we add a config change that adds new metrics targets, usually an integration test is added to check that those metrics get pulled in.

**Comment by philipnrmn** [comment]
I added an integration test per your request :)

File: CHANGES.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * [Marathon] Introduce new exit error code when the framework was removed from Mesos.
 * Ensure Bouncer uses configured web proxy. (DCOS_OSS-5167)
+* Added Fluent Bit metrics to the pipeline (DCOS-54425)
```
Comment (branden) [inline CHANGES.md]: This entry should be added at the top of the section, with a following blank line. See https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines#format

Comment (philipnrmn) [reply CHANGES.md]: ugh, looks like a rebase artefact, thanks for catching it

Comment (philipnrmn) [reply CHANGES.md]: upgefixed

File: CHANGES.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### Fixed and improved
 
+* Added Fluent Bit metrics to the pipeline (DCOS-54425)
```
Comment (jgehrcke) [inline CHANGES.md]: Hey. this should have been added to the 1.13.2 section above.

File: CHANGES.md (hunk 1 above)
Comment (jgehrcke) [reply CHANGES.md]: Thank you @branden for helping out here! :-)

Most importantly, this should have gone into the "in development" section, for 1.13.2 :-) haha... doing that now in an editorial update.

**Pull request merged**